stream_alert_cli/terraform/generate.py

```diff
@@ -163,12 +163,16 @@ def generate_main(**kwargs):
     # Setup Firehose Delivery Streams
     generate_firehose(config, main_dict, logging_bucket)
 
-    # Configure global resources like Firehose alert delivery
+    # Configure global resources like Firehose alert delivery and alerts table
     main_dict['module']['globals'] = {
         'source': 'modules/tf_stream_alert_globals',
         'account_id': config['global']['account']['aws_account_id'],
         'region': config['global']['account']['region'],
-        'prefix': config['global']['account']['prefix']
+        'prefix': config['global']['account']['prefix'],
+        'alerts_table_read_capacity': (
+            config['global']['infrastructure']['alerts_table']['read_capacity']),
```

Comment:
Are these required on purpose? If a user forgot to add them to the `conf`, it would throw `KeyErrors`

Reply:
Yes, they should be required IMO. Dynamo cost is directly related to capacity, and it should be an error if you don't have the table capacity as part of your config.

This requirement could be removed when we add support for autoscaling, for example.